Mathlib/SetTheory/ZFC/Rank.lean

```diff
@@ -64,7 +67,7 @@ variable {x y : PSet.{u}}
 theorem rank_empty : rank ∅ = 0 := by simp [rank]
 
 @[simp]
-theorem rank_insert : rank (insert x y) = max (succ (rank x)) (rank y) := by
+theorem rank_insert (x y : PSet) : rank (insert x y) = max (succ (rank x)) (rank y) := by
```

Comment:
You're making a consecutive block of variables explicit. Why don't you use `variable (x y)`?

Reply:
I'm not generally fond of using `variable` with explicit arguments. It's easy to forget and end up with explicit variables in an `iff` that gets added later, for instance.

Reply:
I see.